Makefile

```diff
@@ -0,0 +1,6 @@
+.PHONY: generate
+generate:
+	cd go && controller-gen paths=./... crd:trivialVersions=true object output:crd:artifacts:config=crd/
+	cd go/generate && go run types.go
+	cp go/crd/*.yaml deploy/helm-chart/kube-mail/crds/
```

Comment:
Maybe _move_ instead of copy? Or just generate them in `deploy/helm-chart/kube-mail/crds` from the beginning? Or at least, add `go/crd/*.yaml` to the `.gitignore`? Right now, they're checked in twice.

Reply:
controller-gen will now output them to `deploy/helm-chart/kube-mail/crds`. I've removed them from the `go` folder.